### DISCUSSION TITLE
My Personal Theme Doesn't Want To Work On Powershell & Ubuntu Bash on WSL2

### DISCUSSION THREAD

**Discussion by inalelub** [JanDeDobbeleer/oh-my-posh#2092] [Troubleshoot] [answered]
Hi, Everyone! 

I recently added oh-my-posh on Ubuntu on my WSL2 and I manually downloaded it and didn't use homebrew for installation. I followed every instruction to the point that it was working perfectly. I used my own personal customized theme which I was using on my Powershell and shared it with WSL Ubuntu using the flag "--config" as stated in the docs. Now after I reloaded my Windows Terminal on my Windows side my original theme got overwritten by the default theme, I assume looking at the json file and my original theme got a new file extension called bak and now my Powershell theme doesn't render properly and also my bash theme doesn't automatically render until I type "exec bash" everytime. 

So, should I paste the "exec bash" command on my ".bashrc" file for it to automatically render each time & for the windows side should I delete my overwritten file of ".ohmyposh.json" or the new ".ohmyposh.json.bak" file but that file has my json theme exactly and if I delete that one then my Ubuntu prompt won't work because it's not pointing to any theme called ".ohmyposh.json" anymore. I got a huge dilemma and as for my Powershell prompt my "$PROFILE" still has my default theme set "Set-PoshPrompt -Theme ~/.ohmyposh.json" so could you please remedy this?

<img width="866" alt="File System" src="https://user-images.githubusercontent.com/69175349/163584399-4a080358-7428-4e67-8081-1979d03f1c2b.png">

<img width="866" alt="oh-my-posh" src="https://user-images.githubusercontent.com/69175349/163584244-c0618daf-ca15-4826-8be0-51d0b6ee9c7b.png">

<img width="866" alt="oh-my-posh-bak" src="https://user-images.githubusercontent.com/69175349/163584420-dd132dd1-8d85-41f3-8122-0a6890e805a9.png">

<img width="872" alt="ubuntu" src="https://user-images.githubusercontent.com/69175349/163584443-0a6ddea8-92c2-4739-98f5-550a835e616d.png">

<img width="866" alt="bashrc" src="https://user-images.githubusercontent.com/69175349/163584468-fcd937f2-c371-4019-90fe-1e22c44784ba.png">

**Comment by JanDeDobbeleer** [comment, marked as answer]
@blkniga make sure everything is on the same version, have a look [here](https://ohmyposh.dev/docs/migrating) to migrate to the latest version.
